Example - Warming - Change image + small fixes (#377)
lethe-cfd/lethe: doc/source/examples/multiphysics/warming-up-a-viscous-fluid/warming-up-a-viscous-fluid.rst

.. note:: 

>     Heat transfer phenomenon occur at a much larger characteristic time than fluid transport phenomenon. To reach a stable state for the system, the end time much be quite big, but the time step can also be increased (in the limit of numerical convergence).

Reviewer: ```suggestion
    Heat transfer phenomenon occur at a much larger characteristic time than fluid transport phenomenon. To reach a stable state for the system, the end time much be quite big, but the time step can also be increased (in the limit of numerical stability).
```

.. note::

>     As the fluid velocity is not influences by heat transfer (one-way coupling), the fluid velocity will remain constant for the whole simulation across the domain, and as heat transfer occurs at a larger scale, the mesh can be coarse.

Reviewer: ```suggestion
    As the fluid velocity is not influenced by heat transfer (one-way coupling), the fluid velocity will remain constant for the whole simulation across the domain, and as heat transfer occurs at a larger scale, the mesh can be coarse.
```

.. image:: images/domain_t7_water_rescale_nodiss.png
    :alt: Rescaled domain with temperature (t = 7)
    :width: 30%

Possibilities for extension
--------------

* Study the **sensitivity to the time step**, namely to assess how large the ``time step`` can be before stating any difference in the heat transfer solution.

> * Test **different time integration scheme** and see if there is any difference in the computational cost and/or the precision with reguards to the analytical solution.
> * See how the **resolution order** (``velocity order``, ``pressure order`` and ``temperature order``) affects the precision with reguards to the analytical solution.

Reviewer 2: ```suggestion
* Test a **different time integration scheme** and see if there is any difference in the computational cost and/or the precision with regards to the analytical solution.
* See how the **resolution order** (``velocity order``, ``pressure order`` and ``temperature order``) affects the precision with regards to the analytical solution.
```
